src/app/utils/open-api-parser.ts

```diff
@@ -0,0 +1,50 @@
+export function parseOpenApiResponse(params: any) {
+  const { options: { paths }, url } = params;
+
+  try {
+    const parameters: any[] = [];
+    const requestUrl = Object.keys(paths)[0];
+    const verbs = Object.keys(paths[`${requestUrl}`]);
+    const pathValues: any = Object.values(paths)[0];
```

Comment:
Typing suggestion from above continued: Without types, the caller is at least expected to read the destructuring here and it is hard to tell what qualifies as valid input at a glance.

We can use types in function signature to help callers in understanding the purpose of the function. Comments are useful if there are other expectations of input that can't be defined as part of the formal signature (i.e. types).